### DISCUSSION TITLE
Selector Icon

### DISCUSSION THREAD

**Discussion by pcho** [tailwindlabs/heroicons#743] [Ideas]
Would be great to bring back Selector Icon in v2.0.

![CF3DB677-ED92-4760-B74B-44304ECBE1C2](https://user-images.githubusercontent.com/41922527/186280166-5eb0522c-f11c-4e7a-b3c0-25fe02aa41a1.jpeg)

**Comment by LarsFlieger** [comment]
Maybe use `arrows-up-down` instead.
![image](https://user-images.githubusercontent.com/47084064/186448671-38b1ca69-a2a1-47e4-b687-17a9a498f09b.png)

**Reply by pcho** [reply]
> Maybe use `arrows-up-down` instead.
> 
> ![image](https://user-images.githubusercontent.com/47084064/186448671-38b1ca69-a2a1-47e4-b687-17a9a498f09b.png)
> 
> 

Nope, that's for sorting.

**Reply by LarsFlieger** [reply]
True...

**Comment by sschoger** [comment]
Hey! The selector icon has now been added back. Sorry I missed that one.
